airbyte-integrations/connectors/source-youtube-analytics/metadata.yaml

```diff
@@ -25,6 +25,11 @@ data:
     oss:
       enabled: true
   releaseStage: beta
+  releases:
+    breakingChanges:
+      1.0.0:
+        message: Version 1.0.0 introduces changes to the report version which changes the stream names.
```

Comment:
```suggestion
        message: Version 1.0.0 introduces new streams that are added instead of old deprecated streams due to change on API side. Users will need to Refresh Source Schema and Clear affected streams. Please, follow Migration Steps in the Migration Guide: https://docs.airbyte.com/integrations/sources/youtube-analytics-migrations. For more info about deprecation see API docs: https://developers.google.com/youtube/reporting/revision_history#bulk-data-reports-for-youtube-analytics. 
```